stores/migrations.go

```diff
@@ -25,6 +25,54 @@ func performMigrations(db *gorm.DB) error {
 			return err
 		}
 	}
+	// If the shards table exists, we add the db_slab_id column to slices and
```

Comment:
one of the things I've learned is that database migrations should have the following:
- backups
- extremely verbose logging in custom logging files
- acid transactions

we are still in beta but don't you think we should maybe add some of the things above, e.g. add a custom log file with every action, switch the db logger's log level and output it to a custom file? essentially make it so that in a worst case scenario we can recover or revert the migration?